package.json

```diff
@@ -16,8 +16,9 @@
     "lint": "eslint src tests",
     "rebuild": "shx rm -rf node_modules && npm cache clean --force && npm install",
     "serve": "serve examples",
-    "start": "node .",
+    "start": "node --experimental-vm-modules .",
```

Comment:
I think this will need mobiles (Quest + Magic Leap) support.

This will probably require a `node` rebuild to `12.5.0`+. The reason we don't have this is that recent node 12 versions introduced android syscall incompatibilities. Either way, we should resolve this before merging.

Reply:
hm, I'm not sure what you mean. Starting node with `--experimental-vm-modules` should work on all versions of node v11 and v12, so no extra work should be necessary.

Reply:
Yeah, I've been using this with node 11 inside [webrun](http://github.com/RangerMauve/webrun)

Reply:
I didn't know this went back to Node 11. Disregard!